ci/fpm-deployment.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+#
+set -ex
+
+DESTDIR="stdlib-fpm"
+
+FYFLAGS="-DMAXRANK=4"
+
+mkdir -p $DESTDIR/src
+mkdir -p $DESTDIR/test
+
+# Preprocess stdlib sources
+ls src/*.fypp | cut -f1 -d. | xargs -i{} fypp {}.fypp {}.f90 $FYFLAGS
```

Comment:
Don't do this, artifacts of this script must never exist in the source directory they were generated from as they possibly interfere with future runs. Also, `-i` is deprecated in `xargs`. Instead, directly write all artifacts in the destination directory.

I'll send a PR against your branch shortly with an updated version of the script.

Reply:
See https://github.com/fortran-fans/stdlib/pull/1